circleguard/wizard.py

```diff
@@ -120,95 +118,198 @@ def __init__(self, parent=None):
 
         layout = QVBoxLayout()
         layout.addWidget(label)
         self.setLayout(layout)
 
 class TutorialPage2(WizardPage):
-    def __init__(self, parent=None):
-        super().__init__(parent)
-        self.setTitle("A Small Tutorial (cont.)")
-        label = QLabel("<p>When you run a check on a map, user, osr files, etc. the results can "
-                       "be confusing to read at first.</p>"
-                       "<p>Circleguard compares replays in sets of two at a time. The similarity "
-                       "reported to you is, roughly speaking, the average pixel distance between the two cursors. "
-                       "Anything below 18 similarity is almost certainly a replay steal, and when a replay that scores "
-                       "below 18 is found, circleguard will audially and visually alert you (OS specific), as well as "
-                       "printing to the center area. This threshold is adjustable in the settings and on each tab.</p>"
-                       "<p>Circleguard will also print results (but not otherwise alert you) for replays that have a "
-                       "similarity under 25 by default. This is to give you suspicious replays that you may want to investiage further. "
-                       "This threshold is, of course, adjustable in the settings.")
-
-        label.setTextFormat(Qt.RichText)
-        label.setTextInteractionFlags(Qt.TextBrowserInteraction)
-        label.setOpenExternalLinks(True)
-        label.setWordWrap(True)
+    def __init__(self):
+        super().__init__()
+        self.setTitle("Tutorial (Loadables)")
+        label = WizardLabel("<p>Circleguard uses five different objects (called Loadables) to represent replays. "
+                "They are Map Replay, Local Replay, Map, User, and All Map Replays by User.</p>"
+                "<p>A Map Replay represents a replay by a single User on a Map.</p>")
+        image = QLabel()
+        # SmoothTransformation necessary for half decent image quality
+        image.setPixmap(QPixmap(str(resource_path("resources/tutorial/map_replay_empty.png"))).scaledToWidth(500, Qt.SmoothTransformation))
+        label2 = WizardLabel("<p>If you wanted cookiezi's replay on Freedom Dive, for instance you would enter 129891 "
+                " as the Map id and 124493 as the User id.</p>"
+                "<p>By default (when Mods is left empty), the highest scoring replay by that user will be used in a Map Replay. "
+                "However, if you specify Mods, the replay with that mod combination will be used instead.<p>"
+                "Mods are specified using a string made up of two letter combinations — enter HDHR for Hidden Hardrock, "
+                "EZ for Easy, NFDT for Nofail Doubletime, etc. The order of the mods does not matter when entering (HDHR is the same as HRHD).<p>")
 
         layout = QVBoxLayout()
         layout.addWidget(label)
+        layout.addWidget(image)
```

Comment:
You mean the literal widget right, not just a pixmap of it?

Yeah it is kinda misleading since it looks like you can type stuff into the boxes but it's just an image. They will have the gui open to play around with though

Reply:
We can do it like this: ![](https://i.imgur.com/L5o1kCF.png)
You can't interact with it but we don't need to store images and it looks sharper

Reply:
Any reason to add the actual widget and not convert it to a pixmap and add that to the label?

Reply:
extra storage, we have to create a new image if we change something, resizing the image is not perfect, themes do not apply,...

Reply:
No no I get why not to use screenshots, you're totally right about that. But why not create the widget and then use https://doc.qt.io/qt-5/qwidget.html#grab or something similar to render it into a pixmap and display that instead of setting these flags to make it unclickable

Reply:
> resizing the image is not perfect, themes do not apply